src/feature/hs/hs_client.c

```diff
@@ -1563,42 +1563,51 @@ hs_config_client_authorization(const or_options_t *options,
 
     hs_client_service_authorization_t *auth = NULL;
     ed25519_public_key_t identity_pk;
+    log_info(LD_REND, "Loading a client authorization key file %s...",
+             filename);
 
-    if (auth_key_filename_is_valid(filename)) {
-      /* Create a full path for a file. */
-      client_key_file_path = hs_path_from_filename(key_dir, filename);
-      client_key_str = read_file_to_str(client_key_file_path, 0, NULL);
-      /* Free the file path immediately after using it. */
-      tor_free(client_key_file_path);
-
-      /* If we cannot read the file, continue with the next file. */
-      if (!client_key_str) {
-        continue;
-      }
+    if (!auth_key_filename_is_valid(filename)) {
+      log_warn(LD_REND, "The filename is invalid.");
```

Comment:
I tried to turn off a client authorization without removing the file by appending `.off` to the file name but then this warning was triggered.

I propose we put it at notice level and we say that we ignored the file instead so the user is aware.